.github/renovate.json

```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
+	"extends": ["github>sapphiredev/readme:sapphire-renovate"],
+	"labels": ["Meta: Dependencies"],
+	"npm": {
+		"packageRules": [
+			{
+				"matchPackagePatterns": [
+					"@napi-rs/canvas",
```

Comment:
Why is this ignored?

Reply:
For the same reason it is ignored in skyra and canvas-constructor. Code doesn't compile with the latest version. Informed you at the time but you're too busy to fix it 💀